src/stories/components/Carousel/templates/Default.html

```diff
@@ -3,7 +3,7 @@
     <div class="carousel-inner">
       <div class="carousel-item active" title="title">
         <a href="https://qld.gov.au/">
-          <img src="https://via.placeholder.com/920x572/808080/808080" alt="placeholder" />
+          <img src="https://static.qgov.net.au/assets/v4/latest/images/placeholder-image.png" alt="placeholder" />
```

Comment:
nice to have: have 920x572 in file name (note that this image is used with a different size, is that ok?)

Reply:
currently https://static.qgov.net.au/assets/v4/latest/images/ only has 1 placeholder image (placeholder-image.png)
I've included other plaecholder size images in this change, but they only can be used after the next release.

btw the image size doens't matter because the css has overridden the size, just it doesn't look perfect as it got streched sometimes

Reply:
since you added the assets, we may want to split this into two, one for getting them loaded (or at least manually added to s3) and another to use them.